src/includes/shape.js

```diff
@@ -503,9 +500,9 @@ pub.strokeWeight = function (weight) {
  *
  * @cat Typography
  * @method objectStyle
- * @param  {PageItem|String} pageItemOrName  A page item whose style to return or the name of the object style to return.
- * @param {Object} [props]  Optional: An object of property name/value pairs to set the style's properties.
- * @return {ObjectStyle}  The object style instance.
+ * @param  {PageItem|String} itemOrName A page item whose style to return or the name of the object style to return.
```

Comment:
Because that is what the argument is called in the function itself. Not sure, if that needs to be consistent, in other places it looked like it was consistent. Should I rename it to `pageItemOrName`? Then it would also go ahead and rename in the function itself, right?